spec/author_fields_spec.rb

```diff
@@ -78,10 +78,11 @@
     it_behaves_like 'display field', :author_person_display
   end
 
-  context 'author_person_full_display' do
-    it_behaves_like 'multi-valued field', :author_person_full_display
-    it_behaves_like 'display field', :author_person_full_display
-  end
+  # We are NOT populating this field, as it causes duplicate author fields to show in the record view
```

Comment:
If we still want to keep these tests commented-out, maybe just add something to the comment that we will use this test when we have more author information to populate the full display?